.changeset/nine-fireants-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+"@opennextjs/cloudflare": patch
+---
+
+fix: CustomRequest instantiation
```

Comment:
```suggestion
fix: fix CustomRequest instantiation sometimes throwing
```

or something like that, what I'm suggesting is there's very little context here on what the change actually is

Reply:
Added more docs/links as a fixup PR.